### ISSUE TITLE
Wine / Wine staging versions far behind

### ISSUE THREAD

**Issue by keybreak** [PhoenicisOrg/phoenicis-winebuild#137]
There's already wine 6.8 released, POL builds are behind currently:

| Branch | Version |
| --- | --- |
Wine | 6.5
Wine staging | 6.3 |

**Comment by plata** [comment]
I do not have a solution for Mac but for Linux #120 would solve this issue.

**Comment by Gcenx** [comment]
> I do not have a solution for Mac but for Linux #120 would solve this issue.

Have you manage to make any progress on this?

I could get functional darwin builds with a minimal set of libraries for a working wine(64/32on64) but it won’t contain everything but that’s not really much different than it is currently.

**Comment by plata** [comment]
I didn't work on this topic (and I don't think I will - at least not in the near future). So anybody who wants to give it a try is more than welcome.

**Comment by keybreak** [comment]
Same old story, far behind :cry:

**Comment by keybreak** [comment]
Is project dead or alive? :laughing:

**Comment by qparis** [comment]
Yes it is :-)

**Comment by keybreak** [comment]
Well...Linux `wine` / `wine-staging` is **super** behind now...

**Comment by qparis** [comment]
Yes, should be quite easy to fix it if you want to help us :) Everything is containerized, so it should be fairly easy. 
Feel free to ask if you need assistance

**Comment by keybreak** [comment]
A little more time and experience should pass as assistance? :laughing: 
Sorry, not sure i can attend it in next half year at least...But i'll take an occasional look
